calendar_resource/README.rst

```diff
@@ -10,11 +10,26 @@ This module adds new features to facilitate resource management with meetings:
 
 * To attach resources to a meeting, go to `Calendar` => Click on a meeting => `Edit` =>
   `Edit` => add any related resources.
-* To edit or add a resource, go to `Settings` => `Resource` => `Resources`.
+* To edit or add a resource, go to `Settings` => `Resource` => `Resources`. Developer mode
+  has to be enabled for this menu to show.
```

Comment:
This module depend on the Calendar app. So one idea is to add it there.
Maybe we could move the "Settings > Technical > Calendar" menu to "Calendar > Configuration", not forgetting to remove the "Technical Features" group from it's Access Rights, so that other people can see it. 
This new "resources" menu should go there.
And I think we need a new "Resource manager" user to control access to these configurations...

Just bringing more work, sorry... 😢